roles/openshift_health_checker/test/search_journalctl_test.py

```diff
@@ -0,0 +1,199 @@
+import pytest
+import search_journalctl
+import sys
+
+from time import time
+
+
+SAVED_TIMESTAMPS = list()
```

Comment:
This looks like a side-effect of a harder to test API.

Make the function pure (it should not call `time.time()`, but take two timestamps as input instead) and it gets easier to test.